grc/CMakeLists.txt

```diff
@@ -25,9 +25,9 @@ include(GrPython)
 message(STATUS "")
 
 GR_PYTHON_CHECK_MODULE_RAW(
-    "python2 >= 2.7.6 or python3 >= 3.6.5"
+    "python3 >= 3.6.5"
```

Comment:
I guess we should be checking for GR_MIN_PYTHON here

Reply:
true.

Reply:
technically `GR_PYTHON_MIN_VERSION` ...

Reply:
Fixed on the other PR.